### PR TITLE
Add CLO mainnet and testnet RPC

### PR DESCRIPTION
**Pull request by yograterol**
- Add RPC node of https://callisto.network/ for Testnet
- Add RPC node of https://callisto.network/ for Mainnet

### CODE REVIEW

File: app/scripts/nodes.js [hunk 1]
```diff
@@ -256,7 +258,31 @@ nodes.nodeList = {
...
+        'blockExplorerTX': 'https://explorer.callisto.network/tx/[[txHash]]',
+        'blockExplorerAddr': 'https://explorer.callisto.network/addr/[[address]]',
+        'type': nodes.nodeTypes.CLOT,
+        'eip155': true,
+        'chainId': 7919,
+        'tokenList': require('./tokens/cloTokens.json'),
+        'abiList': require('./abiDefinitions/cloAbi.json'),
+        'service': 'Callisto.network',
+        'lib': new nodes.customNode('https://clo-testnet3.0xinfra.com/', '')
+    },
```
Comment (akx20000a) [inline app/scripts/nodes.js]: This gives me a syntax error, }, -> }

Comment (akx20000a) [inline app/scripts/nodes.js]: Syntax error } to }, will be fine 😄

Comment (ghost) [inline app/scripts/nodes.js]: Syntax error } to }, will be fine 😄

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by yograterol** [comment]
@akx20000a thanks!

**Review by ghost — APPROVED** [review]
ACK

**Comment by yograterol** [comment]
ACK

**Comment by gamalielhere** [comment]
@yograterol Can you fix the conflict? I'll merge it afterwards

**Review by gamalielhere — APPROVED** [review]

**Comment by yograterol** [comment]
@gamalielhere Rebase done.

**Pull request merged**